features/inventory.feature

```diff
@@ -33,10 +33,44 @@ Scenario: Create a new ProductInformation
     And I press the "create" Button
     Then I should see the message "Success"
 
+Scenario: Query a product by Name
+    When I visit the "Home Page"
+    And I select "prod_name" for "searchKey"
+    And I set the "searchValue" to "iPhone"
+    And I press the "search" Button
+    Then I should see the message "Success"
```

Comment:
Besides the Success message, shouldn't we also see the iPhone entry in the Search Results table too?